docs/fides/docs/guides/generate_resource.md

```diff
@@ -67,3 +67,41 @@ dataset:
 ```
 <!-- TODO: Link to the `annotate dataset` usage documentation below, when it exists. -->
 The resulting file still requires annotating the dataset with data categories to represent what is stored. 
+
+# Working With an AWS Account
+
+The `generate system aws` command can connect to an AWS account and automatically generate resource YAML file based on tracked resources. Authentication is managed through environment variable configuration defined by [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html). 
+
+We can define our credentials directly as environment variables:
+```sh
+export AWS_ACCESS_KEY_ID="<my_access_key_id>"
+export AWS_SECRET_ACCESS_KEY="<my_access_key>"
+export AWS_DEFAULT_REGION="us-east-1"
+```
+
+Or reference a profile through an environment variable:
+```sh
+export AWS_PROFILE="my_profile_1"
+export AWS_DEFAULT_REGION="us-east-1"
+```
+
+Then invoke the `generate system aws` command 
+```sh
+./venv/bin/fidesctl generate system aws \
+  fides_resources/aws_systems.yml
+```
+
+The result is a resource file with a system that represents a redshift cluster defined in our account:
+```yaml
+system:
+- fides_key: my_redshift_cluster
+  organization_fides_key: default_organization
+  name: my_redshift_cluster
+  description: 'Fides Generated Description for Cluster: my_redshift_cluster'
+  meta:
+    endpoint_address: my_redshift_cluster.us-east-1.redshift.amazonaws.com
+    endpoint_port: '5439'
```

Comment:
Thoughts on putting this data in a `fidesctl_meta` instead, and keeping the original `meta` tag to just be ignored by the parser?

There's a similar `fidesops_meta` schema in the fidesops project, you can see the definition here: https://github.com/ethyca/fidesops/blob/main/src/fidesops/schemas/dataset.py#L98

Reply:
This would also allow you to validate these fields using Pydantic schemas, which would be a good idea... right now it looks like there are no validations on this right now

Reply:
we had talked about throwing this into the `meta` section before, but maybe we can leave `meta` as something for the user to just add their own tags (or rename it `tags?`), and `fidesctl_meta` as information we would validate and use?

Reply:
The separation of `fidesctl_meta` and `meta` makes sense to me. 

We should make it clear that the field isn't important to the application. Something like `user_meta` would make that clearer imo.